docs/reference/getting-started.asciidoc

```diff
@@ -158,6 +158,8 @@ On macOS, Elasticsearch can also be installed via https://brew.sh[Homebrew]:
 brew install elasticsearch
 --------------------------------------------------
 
+If installation succeeds, Homebrew will finish by saying that you can start Elasticsearch by entering `elasticsearch`. Do that now. The expected response is described below, under <<successfully-running-node>>.
```

Comment:
It appears I cannot easily make comments in sections that were not changed, so I'm adding it here. I think you need an anchor `[[successfully-running-node]]` later above the section title you are linking to. I added it locally at line 221 (between the [float] marker and the section title). Another small thing if you are editing this file another time anyway: we try to keep the line length a bit shorter (although this is not consistently applied throughout the docs, as demonstrated just a few lines below), so if you don't mind I think this could use a line break maybe after the first sentence. Not a necessary change though.